src/explore-education-statistics-admin/src/pages/release/components/ReleaseSummaryForm.tsx

```diff
@@ -105,7 +90,9 @@ const ReleaseSummaryForm = <
     timePeriodCoverageStartYear: Yup.string()
       .required('Enter a year')
       .length(4, 'Year must be exactly 4 characters'),
-    releaseTypeId: Yup.string().required('Choose a release type'),
+    releaseType: Yup.mixed<FormValues['releaseType']>()
+      .oneOf(Object.keys(releaseTypes) as ReleaseType[])
+      .required('Choose a release type'),
```

Comment:
This validation message wasn't being shown for me, if I didn't select a release type on `ReleaseCreatePage`

Reply:
I was having trouble with this on Friday and it's the reason why there's a couple of failing front-end tests causing the build failure on this PR. I've not been able to sort it yet.

I made `releaseType` optional because I don't want to initialise it with a value but want to force a value to be chosen instead using validation. I suspect it's to do with that.

The docs on `oneOf` say
> Note that undefined does not fail this validator, even when undefined is not included in arrayOfValues. If you don't want undefined to be a valid value, you can use Schema.required.

... but `required` doesn't seem to be working here.

I'm looking for help on this one if anyone knows the solution? 🙂